app/controllers/postquotes_controller.rb

```diff
@@ -0,0 +1,62 @@
+class PostquotesController < ApplicationController
+	def index
+		@postquotes = Postquote.all
+	end
+
+	def new
+		@postquote = Postquote.new
+	end
+
+	def create
+		@new_postquote = Postquote.new(postquote_params)
+		if @new_postquote.save
+			flash[:notice] = "Thanks for submitting a quote!"
+			redirect_to postquote_path(@new_postquote)
+		else
+			flash[:error] = "Please try again"
+			redirect_to new_postquote_path
+		end
+	end
+
+	def edit
+		@postquote = Postquote.find_by(id: params[:id])
+	end
+
+	def update
+		@postquote = Postquote.find_by(id: params[:id])
+		if @postquote.update(postquote_params)
+			flash[:notice] = 'Update to your quote was saved'
+			redirect_to postquote_path(@postquote)
+		else 	
+			flash[:error] = "Please try again"
+			render :edit
+		end
+	end
+
+	def destroy
+		@postquote = Postquote.find_by(id: params[:id])
+		@postquote.destroy
+		redirect_to '/'
+	end
+
+	def show
+		load_wallpaper
```

Comment:
If I were you I'd have your load_wallpaper return the wallpaper_url and use `@wallpaper = load_wallpaper` here - calling methods with side effects can make it harder to figure out what's going on.